src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay/configuration/setup-session-replay.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: "Setup session replay"
+metaDescription: "Enable session replay to play back user interactions and improve page performance."
+freshnessValidatedDate: 2024-12-19
+---
+
+Fully utilize session replay in New Relic, start by enabling this feature in your browser application settings. This step is essential for capturing user interactions on your web app, providing insights into user journeys, and troubleshooting issues effectively. Follow the steps below to enable session replay and ensure your browser agent is configured correctly.
```

Comment:
To fully?

Reply:
Apologies, I did not see this, i have removed the word